<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>572</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Kosuzu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widgetButtons" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>29</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>29</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #505050;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonNotes">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Toggle notes (N)</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonStyle">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Switch note style (Tab)</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonFit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Toggle fit (F)</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRightToLeft">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Toggle right to left</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonFullscreen">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Toggle fullscreen (F11)</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonFirst">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>First page</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPrev">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Previous page (Left arrow)</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPage">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Jump to page (P)</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNext">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Next page (Right arrow)</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonLast">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Last page</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCollection">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Open collection window</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
         <property name="text">
          <string>Collection</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDownloader">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Open downloader window</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #333333;</string>
         </property>
         <property name="text">
          <string>Downloader</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>940</width>
         <height>497</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="imageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenKszFile"/>
    <addaction name="actionOpenProgramFolder"/>
    <addaction name="actionOpenDownloadsFolder"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGitHubRepository"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutKosuzu"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpenKszFile">
   <property name="text">
    <string>Open ksz file</string>
   </property>
  </action>
  <action name="actionCollection">
   <property name="text">
    <string>Collection</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="text">
    <string>Download</string>
   </property>
  </action>
  <action name="actionAboutKosuzu">
   <property name="text">
    <string>About Kosuzu</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionCheckForUpdate">
   <property name="text">
    <string>Check for update</string>
   </property>
  </action>
  <action name="actionGitHubRepository">
   <property name="text">
    <string>GitHub repository</string>
   </property>
  </action>
  <action name="actionOpenProgramFolder">
   <property name="text">
    <string>Open program folder</string>
   </property>
  </action>
  <action name="actionOpenDownloadsFolder">
   <property name="text">
    <string>Open downloads folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
